### PR TITLE
[chore] Add Startup Docs

### PR DESCRIPTION
**Pull request by warittornc**
## Implementation details
- Do 1
- Do 2

## Please ensure the following requirements are met before submitting a pull request:

- [ ] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the GitHub PR explorer)

### CODE REVIEW

File: docs/setup_bothan.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+### Optional: Specify Config File Path
+
+If the configuration file is not located in the default directory `$HOME/.bothan/`, use the --config flag to specify the
+path to the configuration file:
+
+```bash
+bothan start --config /path/to/config.toml
+```
```
Comment (colmazia) [inline docs/setup_bothan.md:52]: Should we add how to setup, run with docker too?

Comment (warittornc) [reply docs/setup_bothan.md:52]: as discussed, not necessary

File: docs/setup_bothan.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```bash 
+cargo install --path bothan-api/server-cli --bin bothan
+```
+
+This will compile and install the bothan executable
+
+## Configuration Setup
+
+Once Bothan is installed, the next step is to initialize the configuration file. By default, the configuration file will
+be created in the `$HOME/.bothan/ directory`.
```
Comment (RogerKSI) [inline docs/setup_bothan.md:27]:
```suggestion
be created in the `$HOME/.bothan/` directory.
```

File: docs/setup_bothan.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+- Initialize the configuration file using the command:
+
+```bash 
+bothan config init
+```
+
+- Open the configuration file in any text editor of your choice and adjust the parameters as necessary
```
Comment (RogerKSI) [inline docs/setup_bothan.md:35]: Do we have a place/document where the user can read the meaning of each config field? 
also, Is there any description for each field in the config file template from the `init `command?

Comment (warittornc) [reply docs/setup_bothan.md:35]: will do in a separate pr

Comment (colmazia) [inline docs/setup_bothan.md:35]: should we state that some sources need api_key for bothan to be able to start

Comment (warittornc) [reply docs/setup_bothan.md:35]: it won't be necessary since the service can be run without the api key

File: docs/setup_bothan.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+- Clone or navigate to the bothan-api repository.
+- Run the following command to build and install the Bothan CLI
+  binary:
+
+```bash 
+cargo install --path bothan-api/server-cli --bin bothan
+```
+
+This will compile and install the bothan executable
```
Comment (RogerKSI) [inline docs/setup_bothan.md:22]: Add more ways to install binary (install/build/download) (example: https://hermes.informal.systems/quick-start/installation.html). can do it in another PR.

**Review by RogerKSI — APPROVED** [review]

File: docs/setup_bothan.md (hunk 4 above)
Comment (warittornc) [reply docs/setup_bothan.md:22]: noted

**Review by colmazia — APPROVED** [review]
